docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM phusion/baseimage:0.10.1
+
+ARG GZV
+
+CMD ["/sbin/my_init"]
+
+RUN apt-get update && \
```

Comment:
I was doing that during development, and it's definitely quite useful. However, RUN commands also create new layers and increase the size of the build as a result. So I removed them afterward to improve performance as recommended here https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers

Should we leave different RUN commands for each instruction on the Dockerfile we release to users? I think this would also increase build time the first time they run a particular build.